docs/reference/ml/ml-shared.asciidoc

```diff
@@ -1113,6 +1113,27 @@ The field used to segment the analysis. When you use this property, you have
 completely independent baselines for each value of this field.
 end::partition-field-name[]
 
+tag::per-partition-categorization[]
+Settings related to how categorization interacts with partition fields.
+end::per-partition-categorization[]
+
+tag::per-partition-categorization-enabled[]
+If true, categorization is done separately for each value of the
+`partition_field_name` field. The default value is false. This setting can be
+set only when you create the job; it cannot be updated. This setting may only be
```

Comment:
> This setting may only be true if...

This sentence makes me wonder about things like this:
- Is there a validation check that generates an error if this condition isn't met?
- Does this mean that if you have a detector where the by_field_name or over_field_name contains `mlcategory` you *must* also specify a partition_field_name?  Or does this condition only apply if you already have a partition_field_name in that detector?
Depending on the answers to those questions, it might be clearer like this:
> To enable this setting, you must also set the `partition_field_name` property to the same value in every detector that uses the keyword `mlcategory`. Otherwise, job creation fails.